tests/cli/test_main_config.py

```diff
@@ -281,3 +285,71 @@ def test_config_env_does_not_exist(
         conda_cli(
             "config", "--get", "channels", "--prefix", "ireallydontexist", "--json"
         )
+
+
+@pytest.mark.parametrize("is_json", [True, False], ids=["json_true", "json_false"])
```

Comment:
I wouldn't include ids here. My rule is to only include ids if it is hard to figure out which test case is failing.

But if we are first using ids I think we should use `pytest.param` to keep the mapping to a minimum:

```suggestion
@pytest.mark.parametrize(
    "is_json",
    [
        pytest.param(True, id="is_json=True"),
        pytest.param(False, id="is_json=False"),
    ]
)
```

Reply:
I'll just get rid of the IDs. It was an auto suggestion by co-pilot that I thought was worth keeping.